### PR TITLE
ScheduleForm: Add `Send Report Now` button

### LINKED ISSUE #133: "Send Report now" button under Schedule
## Is your feature request related to a problem? Please describe.
A button to send the scheduled report now within ScheduleForm.php!

## Describe the solution you'd like
If you press the button the report gets send.

## My dirty hack
```
vim /usr/share/icingaweb2/modules/reporting/library/Reporting/Web/Forms/ScheduleForm.php

# add on line 9:
use Icinga\Module\Reporting\Actions\SendMail;

# add under line 135...:
if ($this->id !== null) {
...

            /**@var FormSubmitElement $sendButton */
            $sendButton = $this->createElement('submit', 'send', [
                'label'          => 'Send Report now',
                'class'          => 'btn-send',
                'formnovalidate' => true
            ]);
            $this->registerElement($sendButton);
            $this->getElement('submit')->getWrapper()->prepend($sendButton);
            if ($sendButton->hasBeenPressed()) {
                $values = $this->getValues();
                $sendMail = new SendMail();
                $sendMail->execute($this->report, $values);
                echo "Report send";
            }
}
```

### PR DESCRIPTION
**Pull request by jhoxhaa**
resolves #133

### CODE REVIEW

**Comment by yhabteab** [comment]
> Can't see `send report now` button here :)

My bad! This is not the right form! 🙈

**Comment by yhabteab** [comment]
## Blocked by

- https://github.com/Icinga/icingaweb2-module-reporting/pull/167

File: library/Reporting/Web/Forms/ScheduleForm.php
Comment (yhabteab) [inline library/Reporting/Web/Forms/ScheduleForm.php:146]: You should use `$this->translate(Send Report Now)` like the PR title and not the action name.

File: library/Reporting/Web/Forms/ScheduleForm.php
Comment (yhabteab) [inline library/Reporting/Web/Forms/ScheduleForm.php:183]:
```suggestion
         $values = $this->getValues();
        if ($this->getPopulatedValue('send')) {
            $action = new $values['action']();
            $action->execute($this->report, $values);

            Notification::success('Report sent successfully');

            return;
        }
```

**Review by yhabteab — CHANGES_REQUESTED** [review]

**Review by yhabteab — APPROVED** [review]

**Review by yhabteab — APPROVED** [review]

**Comment by yhabteab** [comment]
@lippserd Do you want to have a look?

File: library/Reporting/Web/Forms/ScheduleForm.php
Comment (lippserd) [inline library/Reporting/Web/Forms/ScheduleForm.php:176]: Please replace `t()` with `$this->translate()`. After that it's free to merge.

**Review by lippserd — APPROVED** [review]

**Pull request merged**
